Lib/test/test_pickletools.py

```diff
@@ -7,6 +7,10 @@
 
 class OptimizedPickleTests(AbstractPickleTests, AbstractPickleModuleTests):
 
+    # pickletools.optimize only works in-memory on pickle strings: it
+    # therefore is ok to include large objects inside a large frame.
+    frameless_blobs = False
```

Comment:
I think it would be better to write frameless blobs in `pickletools.optimize()`. This will decrease a peak memory on unpickling.

Reply:
I will try to do that now.